x-pack/test_serverless/api_integration/test_suites/common/alerting/summary_actions.ts

```diff
@@ -63,32 +64,46 @@ export default function ({ getService }: FtrProviderContext) {
       'kibana.version',
       'kibana.alert.consecutive_matches',
     ];
+    const svlCommonApi = getService('svlCommonApi');
+    const svlUserManager = getService('svlUserManager');
+    const supertestWithoutAuth = getService('supertestWithoutAuth');
+    let roleAuthc: RoleCredentials;
+    let internalReqHeader: InternalRequestHeader;
+
+    before(async () => {
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+      internalReqHeader = svlCommonApi.getInternalRequestHeader();
+    });
 
     afterEach(async () => {
       await supertest
```

Comment:
should this be `supertestWithoutAuth`?

Reply:
see: https://github.com/elastic/kibana/pull/183381#discussion_r1606716240

Reply:
https://github.com/elastic/kibana/pull/183381/commits/a0c440b9548ddf95a307dabc89bbce536d35de74